.github/workflows/build.yml

```
@@ -94,7 +94,7 @@ jobs:
     needs: [create_release]
     steps:
     - name: download artifact
-      uses: actions/download-artifact@v1
+      uses: actions/download-artifact@v4
```

If we want to pin this as well:
```suggestion
      uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
```